src/sortStudents.ts

```diff
@@ -1,16 +1,68 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
+
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function averageGrades(grades: number[]): number {
+  return grades
+    .reduce((sum:number, grade: number) => (sum + grade), 0) / grades.length;
```

Comment:
you might not specify types here, TS knows about this types
```suggestion
    .reduce((sum, grade) => (sum + grade), 0) / grades.length;
```